tests/Microsoft.Identity.Web.Test/Base64UrlHelpersTests.cs

```diff
@@ -92,5 +92,28 @@ public void DecodeToBytes_InvalidBase64UrlStringLength_ThrowsException()
             var exception = Assert.Throws<ArgumentException>(decodeAction);
             Assert.Equal(IDWebErrorMessage.InvalidBase64UrlString + " (Parameter 'str')", exception.Message);
         }
+
+        [Fact]
+        public void EncodeString_TakesStringArgument_ReturnsEncodedString()
+        {
+            // Arrange
+            var input = "Hello, world!";
+
+            // Act
+            string? result = Base64UrlHelpers.EncodeString(input);
+
+            // Assert
+            Assert.NotEmpty(result!);
+        }
+
+        [Fact]
+        public void DecodeBytes_TakesNullArgument_ReturnsNull()
+        {
+            // Arrange & Act
+            var result = Base64UrlHelpers.DecodeBytes(null);
```

Comment:
Any reason why we don't use Token.Base64UrlEncoder.Decode? (from Wilson)

Reply:
Yes, we should. will do in later PR.